.github/pull_request_template.md

```diff
@@ -1,39 +1,33 @@
 ## Description
+<!--- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-A detailed explanation of the changes in your PR.
-
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
 
 ## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
-
-If any of these don't apply, please comment below.
+All items *must be* checked:
+- [ ] CI results are inspected
+- [ ] CHANGELOG is updated **OR** not needed
+- [ ] Documentation PR is created and linked above **OR** not needed
 
-## Testing Performed
+## Testing
```

Comment:
I highly appreciate that you kept `How I validated my change` section and that we'll have it together with this one. Due to deep spiritual reasons, I suggest a slight titles tweak. Also, markdown linters like a blank line after any heading.

```suggestion
## Testing

### Automated testing

```

What the checklist tells is the state of automated tests, hence `Automated testing` subsection.
Another subsection is `How I validated my change` and it's also part of `Testing`.

Reply:
My motivation was to strip out everything that's not essential to avoid "template fatigue". So to keep the balance, I'll remove the `## Checklist` header 😛.

Reply:
Maybe titles can be degraded but kept: `##` -> `###` or even `####`?

Reply:
good idea, let's try this out